pkg/openshift/certgen/files.go

```diff
@@ -4,15 +4,54 @@ import (
 	"bytes"
 	"crypto/rand"
 	"encoding/base64"
+	"os"
 	"regexp"
 	"strings"
 	"text/template"
 
-	"github.com/Azure/acs-engine/pkg/openshift/certgen/templates"
+	templ "github.com/Azure/acs-engine/pkg/openshift/certgen/templates"
 	"github.com/Azure/acs-engine/pkg/openshift/filesystem"
 	"golang.org/x/crypto/bcrypt"
 )
 
+type modeinfo struct {
+	path  *regexp.Regexp
+	mode  os.FileMode
+	user  string
+	group string
+}
+
+// These ownerships are processed in order
+var ownerships = []modeinfo{
+	{path: regexp.MustCompile("etc/etcd/.*\\.key$"), mode: 0600, user: "etcd", group: "etcd"},
```

Comment:
use raw quotes to avoid \\\\, anchor the regular expressions with ^
```
{path: regexp.MustCompile(`^etc/etcd/.*\.key$`), mode: 0600, user: "etcd", group: "etcd"},
```

Reply:
Thanks, TIL that back ticks are real quotes.